src/components/button/Button.TypeTest.tsx

```diff
@@ -0,0 +1,81 @@
+// eslint-disable @typescript-eslint/ban-ts-comment
+
+import React from 'react';
+
+import { ShareIcon } from '../icons';
+
+import Button from './index';
+
+const _Button = () => {
+  return (
+    <div>
+      {/* @ts-expect-error children is missing */}
+      <Button onClick={() => null} />
+      {/* @ts-expect-error href not allowed */}
+      <Button href="foo/bar" onClick={() => null}>
```

Comment:
As far as I know this should be allowed. Before was allowed because on the case of tracking